src/perception/vision_pipeline/vision_pipeline/node_detector.py

```diff
@@ -139,7 +139,7 @@ def callback(self, colour_msg: Image, colour_camera_info_msg: CameraInfo, depth_
 
         start: float = time.time() # begin a timer
 
-        colour_frame: np.ndarray = cv_bridge.imgmsg_to_cv2(colour_msg)
+        colour_frame: np.ndarray = cv_bridge.imgmsg_to_cv2(colour_msg, desired_encoding='bgra8')
```

Comment:
this will still work with the actual camera right

Reply:
yep the default arg is `bgra8` but the sim wanted to override with UA8 or smth